### DISCUSSION TITLE
Nzyme-Tap for ARMHF?

### DISCUSSION THREAD

**Discussion by mavacpjm** [nzymedefense/nzyme#999] [General]
I just upgraded to 2.0 from 1.2.2 and I noticed the tap model, i was running node and monitor on same rpi3 on 1.2.2. 

I notice the nzyme-tap is only arm64 is there a armhf version of nzyme-tap ?

**Comment by lennartkoopmann** [comment]
Hi! There are currently no packages for `armhf`. Is there a reason you need it? Raspberry Pi3 runs RPI OS 64bit / arm64 perfectly.

**Reply by mavacpjm** [reply]
I installed 1.2.2 on 32bit and it has a massive bandit datastore and is super well tuned. Was trying to avoid an entire rebuild, I attempted to upgrade but in the process it did some database changes for 2.0 that could not be reversed so when I downgraded back to 1.2.2 I could not restore operating 1.2.2 as it did not recognize some of the db schema it appears. So I was trying to drop the tap on the existing to avoid a rebuild, thanks for the reply!

**Reply by lennartkoopmann** [reply]
Yes, there is no upgrade path to v2.0 because pretty much everything changed. I can help explain the new bandit structure if you want to write a script to move them over.
